contrib/net/http/roundtripper.go

```diff
@@ -58,11 +64,26 @@ func (rt *roundTripper) RoundTrip(req *http.Request) (res *http.Response, err er
 		span.SetTag("http.errors", err.Error())
 	} else {
 		span.SetTag(ext.HTTPCode, strconv.Itoa(res.StatusCode))
+		span.SetTag("network.destination.ip", httpTraceResult.remoteIP)
+		span.SetTag("network.destination.port", httpTraceResult.remotePort)
+		span.SetTag("http.content_type", res.Header.Get("Content-Type"))
```

Comment:
Have you seen stats like this in any other language? If yes, can you please point me to them? I'm trying to make sure that `http.*` is the right namespace for all of them or if we need something like `http_trace.*` or similar.

Reply:
PHP does this a lot under the `curl` namespace :)